### ISSUE TITLE
Weird 'clipping' bug on Chrome

### ISSUE THREAD

**Issue by jpasholk** [rsms/inter#96]
**Describe the bug**

Glyphs have an odd clipping issue or something, not sure how to describe it.

**To Reproduce**

1. Add Inter Ui to a project.
2. use 
`@supports (font-variation-settings: normal) {
  html { font-family: 'Inter UI var alt', sans-serif; }
}`
3. Open in Chrome 70.0.3538.102.

**Expected behavior**

The font should display as intended, without clipping bugs.

**Screenshots**

![screen shot 2018-12-01 at 1 47 00 pm](https://user-images.githubusercontent.com/28698431/49333262-5c640080-f570-11e8-9c57-592a1744eba4.png)

**Environment**

- OS: [e.g. macOS Sierra 10.12.6] - (13-inch, Mid 2010) - NVIDIA GeForce 320M 256 MB
- Chrome 70.0.3538.102
- Not really sure what version. I didn't select one, am just using the font via cdn.

**Additional context**

[CodePen](https://codepen.io/jpasholk/pen/MzBENw)

My fix was I just stopped using the @supports rule.

Let me know if there is anything else I can do to help. This is probably an issue with my super old machine, but you asked me to submit an issue, so here it is!

🍻 

Jpasholk 😄

**Comment by rsms** [comment]
TL;DR: Bug in Chromium

I've heard from someone at the Chromium team that they are aware of this bug. Not sure if/where it's tracked, but this is a bug in Chromium/Chrome with variable fonts. The bug manifests slightly different on Windows and macOS as well.

See [this list of current font rendering issues being worked on by the chromium team](https://bugs.chromium.org/p/chromium/issues/list?can=2&q=font%20render%20status=Assigned&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Component%20Status%20Owner%20Summary%20OS%20Modified)

**Comment by dashkevych** [comment]
It seems like the same issue appears in Firefox (70.0.1)

![clipping issue in firefox](https://i.snipboard.io/VfUCjN.jpg)

**Comment by doutatsu** [comment]
I am seeing this issue, but I don't think it's a bug with the Chrome. I have 2 pages up, both using the font, with the same text, both on Chrome, but one have this issue, while the other doesn't...
